### ISSUE TITLE
Journal, guild ranks, bounty, and other effects reset upon logout.

### ISSUE THREAD

**Issue by Za-Warudo** [TES3MP/TES3MP#194]
Normally, this wouldn't be much of a problem. However, in the current public server, I acquired the Yagrum Bagarn version of Wraithguard (The one that permanently damages health 200 points upon first equip) and when I equipped it after relogging, it damaged my health once more.

**Comment by davidcernat** [comment]
Well, I thought we were pretty clear that functionality would only be present in the next version...

**Comment by Za-Warudo** [comment]
Apologies, I couldn't (and still can't) find anything on the GitHub page regarding it.

Sorry if this has been asked before, but is there a release window for the next version?

**Comment by davidcernat** [comment]
Edit: Sometime in May.

**Comment by davidcernat** [comment]
The scope of this particular issue is too sweeping. Looking at it again, it seems like it potentially encompasses everything in state saving and loading, including journals, factions, bounties, dialogue topics, active effects and script variables.

As a result, I've decided to close it and maybe have separate issues for each area.

That being said, journal state was added with TES3MP/PluginExamples@c02c5817a425229ba06e0fcc5083de3e65e06db3, faction state was added with TES3MP/PluginExamples@99e1def97f30a56e8c31bdbee0c6c29659732c08, bounty state was added with TES3MP/PluginExamples@8970b9e01ffab5aaf6855375376fb6e2e1f15089 and topic state was added with TES3MP/PluginExamples@2b595da5ee39091741314a105e656a3f11f91326

The rest will be part of a future version.
